lib/chef/knife/tidy_server_report.rb

```diff
@@ -62,7 +62,7 @@ def run
             end
           end
 
-          stale_nodes_hash = {'threshold_days': node_threshold, 'count': stale_nodes.count, 'list': stale_nodes}
+          stale_nodes_hash = {'threshold_days': node_threshold, 'node_count': node.count, 'count': stale_nodes.count, 'list': stale_nodes}
```

Comment:
@MarkGibbons this should be `nodes.count`, not `node.count`, right?

Just a small nit-pick, can we change the 'node_count' key to `org_total_node_count` to make it clearer what the number is?

If you can get those two adjustments made, I'll merge this in! 
Thank you!!